test/access/AccessControl.behavior.js

```diff
@@ -210,8 +211,293 @@ function shouldBehaveLikeAccessControlEnumerable(errorPrefix, admin, authorized,
+          account: newDefaultAdmin,
+        });
+      });
+
+      it('accepts a transfer resetting pending default admin and delayed until', async function () {
+        await this.accessControl.acceptDefaultAdminTransfer({ from });
+        expect(await this.accessControl.defaultAdminTransferDelayedUntil()).to.be.bignumber.equal(web3.utils.toBN(0));
+        expect(await this.accessControl.pendingDefaultAdmin()).to.equal(ZERO_ADDRESS);
+      });
+    });
+
+    it('should revert if caller is not pending default admin', async function () {
+      await time.increaseTo(correctIncreaseTo);
+      await expectRevert(
+        this.accessControl.acceptDefaultAdminTransfer({ from: other }),
+        `${errorPrefix}: can't accept defaultAdmin`,
+      );
+    });
+
+    describe('delayedUntil not met', async function () {
+      let incorrectIncreaseTo;
+
+      beforeEach(async function () {
+        incorrectIncreaseTo = correctIncreaseTo.subn(1);
+      });
+
+      it('should revert if block.timestamp is equal to delayed until', async function () {
+        await time.increaseTo(incorrectIncreaseTo);
+        await expectRevert(
+          this.accessControl.acceptDefaultAdminTransfer({ from: correctPendingDefaultAdmin }),
+          `${errorPrefix}: delayedUntil not met`,
+        );
+      });
+
+      it('should revert if block.timestamp is less to delayed until', async function () {
+        await time.increaseTo(incorrectIncreaseTo.subn(1));
+        await expectRevert(
+          this.accessControl.acceptDefaultAdminTransfer({ from: correctPendingDefaultAdmin }),
+          `${errorPrefix}: delayedUntil not met`,
+        );
+      });
+    });
+  });
+
+  describe('cancel transfer default admin', async function () {
+    beforeEach(async function () {
+      await this.accessControl.beginDefaultAdminTransfer(newDefaultAdmin, { from: defaultAdmin });
+    });
+
+    it('resets pending default admin and delayed until', async function () {
+      await this.accessControl.cancelDefaultAdminTransfer({ from: defaultAdmin });
+      expect(await this.accessControl.defaultAdminTransferDelayedUntil()).to.be.bignumber.equal(web3.utils.toBN(0));
+      expect(await this.accessControl.pendingDefaultAdmin()).to.equal(ZERO_ADDRESS);
+    });
+
+    it('reverts if called by non default admin accounts', async function () {
+      await expectRevert(
+        this.accessControl.cancelDefaultAdminTransfer({ from: other }),
+        `${errorPrefix}: account ${other.toLowerCase()} is missing role ${DEFAULT_ADMIN_ROLE}`,
+      );
+    });
+  });
+
+  describe('renouncing admin', async function () {
+    let correctIncreaseTo;
+    let from = defaultAdmin;
+
+    beforeEach(async function () {
+      correctIncreaseTo = (await time.latest()).add(delay).addn(1);
+      await this.accessControl.beginDefaultAdminTransfer(ZERO_ADDRESS, { from });
+    });
+
+    describe('caller is default admin and delayed until is met', async function () {
```

Comment:
I find the phrase "delayed until is met" very hard to parse.

```suggestion
    describe('caller is default admin and delay has passed', async function () {
```

Reply:
Fixed in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4009/commits/14994be0e20da5b1a19046f0f435cdd304583dc1